project/recursive_sorting.py

```diff
@@ -61,7 +61,7 @@ def quick_sort(arr):
         else:
             high.append(num)
 
-    return [quick_sort(low), pivot, quick_sort(high)]
+    return [*quick_sort(low), pivot, *quick_sort(high)]
```

Comment:
cool easy way to build this list! :+1:

Reply:
Yeah I thought it was cool lol, basically a spread operator for python